esp32/sdkconfig.h

```diff
@@ -36,7 +44,6 @@
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK 0
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK_CPU1 0
 
-#define CONFIG_FREERTOS_UNICORE 1
```

Comment:
> Try adding -Wno-error=array-bounds to CFLAGS for the dport_access.c file.

Yes, this worked, with `CONFIG_FREERTOS_UNICORE` restored to 1, basic tests passed.

Wifi works well, per the `towel.blinkenlights.nl` telnet test.

@dpgeorge : Should I update this PR with those changes?